app/js/components/LabResultEntry.jsx

```diff
@@ -478,7 +478,7 @@ export class LabResultEntry extends PureComponent {
                           id="app.labResultEntry.urgencylabel"
                           defaultMessage="Urgency" />
 :&nbsp;
-                        <span className="test-details">{location.state.urgency}</span>
+                        <span className="test-details">{location.state.urgency.charAt(0).toUpperCase() + location.state.urgency.slice(1).toLowerCase()}</span>
```

Comment:
@jomadoye  we'd really like to display "Routine"  and "STAT"  (ie, STAT is capitalized but Routine isn't... this tends to be the preferred capitalization)

"urgency" is a just an enum with three potential values:

https://github.com/openmrs/openmrs-core/blob/master/api/src/main/java/org/openmrs/Order.java#L41

so we can just map as follows:

"ROUTINE" -> "Routine"
"STAT" -> "STAT"
"ON_SCHEDULED_DATE" -> "Scheduled"